py/selenium/webdriver/common/actions/input_device.py

```diff
@@ -39,5 +39,5 @@ def add_action(self, action):
     def clear_actions(self):
         self.actions = []
 
-    def create_pause(self, duraton=0):
+    def create_pause(self, duration=0):
```

Comment:
we should probably deprecate (and display a warning) the misspelled keyword arg here rather than removing it... and then add the new one.   This changes a public API and will break any code that is currently using the misspelled version.